### DISCUSSION TITLE
After upgrade to version 2.7.3 KeePassXC not opening

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8691] [Q&A] [answered]
Hello Team,

After upgrading to version 2.7.3 on macOS Ventura.
Both Intel and Silicone installs crash when unlocking the Database,

Is Ventura OS compatible with the new release ?

I have the logs available.

Thanks

**Comment by droidmonkey** [comment, marked as answer]
2.7.4 will fix this, disable quick unlock for now
